### PR TITLE
Add docs for robots.txt config

### PR DESCRIPTION
**Pull request by mrkschan**

### CODE REVIEW

**Comment by Crell** [comment]
Is there a way to do so declaratively in the yaml files, or via the UI?

**Comment by mrkschan** [comment]
Only via CLI right now.

File: overview/web-ui/configure-environment.md [hunk 1]
```diff
@@ -63,3 +63,11 @@ environment.
 
 You can also manage access to users on multiple environments using the
 project configuration screen.
+
+## Robots.txt
+
+By default, Platform.sh returns a restrictive `robots.txt` on all environments. If you need to provide a custom `robots.txt`, first disable the default one by [Platform.sh CLI](/user_guide/overview/cli/index.html) command below.
+
+```
+platform environment:info -p <project> -e <environment> restrict_robots false
```
Comment (pjcdawkins) [inline overview/web-ui/configure-environment.md]: You could omit `-p <project> -e <environment>` - they are detected based on your current directory. I think we don't include those for CLI commands elsewhere in the docs.

**Comment by mrkschan** [comment]
updated

**Pull request merged**